File: check_braces.js
const fs = require('fs');
const content = fs.readFileSync('src/components/Dashboard.jsx', 'utf8');
let depth = 0;
let lines = content.split('\n');
for (let i = 0; i < lines.length; i++) {
    for (let j = 0; j < lines[i].length; j++) {
        if (lines[i][j] === '(') depth++;
        if (lines[i][j] === ')') depth--;
    }
}
console.log("Parenthesis depth:", depth);

File: check_braces.py
with open('src/components/Dashboard.jsx', 'r') as f:
    text = f.read()

paren = 0
brace = 0
for char in text:
    if char == '(': paren += 1
    if char == ')': paren -= 1
    if char == '{': brace += 1
    if char == '}': brace -= 1

print(f"Paren: {paren}, Brace: {brace}")
